test/AutoRest.TestServer.Tests/body-complex/Generated/Models/Sawshark.Serialization.cs

```diff
@@ -1,7 +1,6 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using System;
```

Comment:
Which generator change fixed this?

Reply:
It wasn't a generator change. Unfortunately, we commit generated code (and don't mark it generated yet (which we have an issue to do)). However, I told resharper to clean up my unused usings, and it hit the generated files. I can revert.